config.json.example

```diff
@@ -0,0 +1,3 @@
+{
+    "discord_token": "asbdhjgasfgadsjhfgasdjfgkhsdjafgfa"
```

Comment:
instead of a random string of letters, perhaps make it either something more meaning full like `DISCORD-TOKEN` or a dummy token in the same format of a real token.

Reply:
ok